### ISSUE TITLE
[BUG] Injection warning

### ISSUE THREAD

**Issue by pavog** [TheFizzWare/DramaticDoors#16]
**Important Information**
Minecraft Version: **1.18.2**
Fabric Version: **1.18.2 / 0.14.6**
DramaticDoors Version: **[DramaticDoors-FABRIC-1.18.2-1.1.jar](https://www.curseforge.com/minecraft/mc-mods/dramatic-doors/files/3811603)**

**Describe the bug**
I created a brand new Fabric 1.18.2 server, installed fabric api and DramaticDoors (DramaticDoors-FABRIC-1.18.2-1.1.jar / Uploaded
May 31, 2022).
When I start the server it throws some warnings and writes them to my server log.
However the server does not crash.

Warnings in my console / log:

```
[13:02:07] [Worker-Main-3/WARN]: Injection warning: LVT in net/minecraft/class_14::method_58(Lnet/minecraft/class_1922;Lnet/minecraft/class_2338;)Lnet/minecraft/class_7; has incompatible changes at opcode 69 in callback dramaticdoors.mixins.json:LandPathNodeMakerMixin from mod dramaticdoors->@Inject::injectDoorType(Lnet/minecraft/class_1922;Lnet/minecraft/class_2338;Lorg/spongepowered/asm/mixin/injection/callback/CallbackInfoReturnable;Lnet/minecraft/class_2680;Lnet/minecraft/class_2248;Lnet/minecraft/class_3614;)V.
--
81 | Expected: [Lnet/minecraft/class_2680;, Lnet/minecraft/class_2248;, Lnet/minecraft/class_3614;]
82 | Found: []
83 | Available: []

84 | [13:02:07] [Worker-Main-3/WARN]: Injection warning: LVT in net/minecraft/class_14::method_58(Lnet/minecraft/class_1922;Lnet/minecraft/class_2338;)Lnet/minecraft/class_7; has incompatible changes at opcode 535 in callback dramaticdoors.mixins.json:LandPathNodeMakerMixin from mod dramaticdoors->@Inject::injectDoorType(Lnet/minecraft/class_1922;Lnet/minecraft/class_2338;Lorg/spongepowered/asm/mixin/injection/callback/CallbackInfoReturnable;Lnet/minecraft/class_2680;Lnet/minecraft/class_2248;Lnet/minecraft/class_3614;)V.
85 | Expected: [Lnet/minecraft/class_2680;, Lnet/minecraft/class_2248;, Lnet/minecraft/class_3614;]
86 | Found: []
87 | Available: []

88 | [13:02:07] [Worker-Main-3/WARN]: Injection warning: LVT in net/minecraft/class_14::method_58(Lnet/minecraft/class_1922;Lnet/minecraft/class_2338;)Lnet/minecraft/class_7; has incompatible changes at opcode 612 in callback dramaticdoors.mixins.json:LandPathNodeMakerMixin from mod dramaticdoors->@Inject::injectDoorType(Lnet/minecraft/class_1922;Lnet/minecraft/class_2338;Lorg/spongepowered/asm/mixin/injection/callback/CallbackInfoReturnable;Lnet/minecraft/class_2680;Lnet/minecraft/class_2248;Lnet/minecraft/class_3614;)V.
89 | Expected: [Lnet/minecraft/class_2680;, Lnet/minecraft/class_2248;, Lnet/minecraft/class_3614;]
90 | Found: []
91 | Available: []
```

**To Reproduce**

- Download Fabric 1.18.2 / 0.14.6
- Download Fabric API
- Download DramaticDoors Fabric
- Start the server
- See the log

**Expected behavior**
No warnings ;-)

Full error log:
https://mclo.gs/H75bpGT

**Comment by Kitteh6660** [comment]
This is something I have been trying to figure out. I will poke around and see if I could look into getting it fixed properly.

**Comment by Kitteh6660** [comment]
Should be fixed with version 1.1.1 of Dramatic Doors Fabric.

**Comment by pavog** [comment]
Yes, it's fixed. Thank you! :-)
